### DISCUSSION TITLE
phase

### DISCUSSION THREAD

**Discussion by kjz1997** [gmtsar/gmtsar#813] [Q&A]
Hi everyone. I would like to know how to evaluate the quality of the interference phase.

**Comment by Xiaohua-Eric-Xu** [comment]
A couple of things to check.
1) Coherence of the interferogram.
2) Connected components from phase unwrapping.
3) How colorful (atmospheric noise) your interferogram is :).

**Reply by kjz1997** [reply]
Thank you for your reply, professor. Do you have any recommended papers? I would like to further understand

**Reply by Xiaohua-Eric-Xu** [reply]
There are a lot of them. A classic one is this one: Lohman, R. B., & Simons, M. (2005). Some thoughts on the use of InSAR data to constrain models of surface deformation: Noise structure and data downsampling. Geochemistry, Geophysics, Geosystems, 6(1).
